editor.planx.uk/src/components/EditorNavMenu.tsx

```diff
@@ -71,62 +73,77 @@ const TooltipWrap = styled(({ className, ...props }: TooltipProps) => (
 }));
 
 const MenuButton = styled(IconButton, {
-  shouldForwardProp: (prop) => prop !== "isActive",
-})<{ isActive: boolean }>(({ theme, isActive }) => ({
-  color: theme.palette.text.primary,
-  width: "100%",
-  border: "1px solid transparent",
-  justifyContent: "flex-start",
-  borderRadius: "3px",
-  "&:hover": {
-    background: "white",
-    borderColor: theme.palette.border.light,
-  },
-  ...(isActive && {
-    background: theme.palette.common.white,
+  shouldForwardProp: (prop) => prop !== "isActive" && prop !== "disabled",
```

Comment:
As this list of excluded props grows, it can be a bit easier to manage these as an array.

```suggestion
  shouldForwardProp: (prop) => !["isActive", "disabled"].includes(prop)
```